outputs/stan.go

```diff
@@ -0,0 +1,42 @@
+package outputs
+
+import (
+	"encoding/json"
+	"log"
+	"strings"
+
+	"github.com/falcosecurity/falcosidekick/types"
+	stan "github.com/nats-io/stan.go"
+)
+
+// StanPublish publishes event to NATS Streaming
+func (c *Client) StanPublish(falcopayload types.FalcoPayload) {
+	nc, err := stan.Connect(c.Config.Stan.ClusterID, c.Config.Stan.ClientID, stan.NatsURL(c.EndpointURL.String()))
+	if err != nil {
+		go c.CountMetric("outputs", 1, []string{"output:stan", "status:error"})
+		c.Stats.Stan.Add(Error, 1)
+		c.PromStats.Outputs.With(map[string]string{"destination": "stan", "status": Error}).Inc()
+		c.Stats.Stan.Add("total", 1)
```

Comment:
This should be before line #14, as we increment in anycase.

Reply:
it is in line 41, but i will make just one